dialogue-core/src/main/java/com/palantir/dialogue/core/ExponentialBackoffStrategy.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * (c) Copyright 2020 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.dialogue.core;
+
+import java.time.Duration;
+import java.util.function.DoubleSupplier;
+
+final class ExponentialBackoffStrategy implements BackoffStrategy {
+
+    private final Duration backoffSlotSize;
+    private final DoubleSupplier random;
+
+    ExponentialBackoffStrategy(Duration backoffSlotSize, DoubleSupplier random) {
+        this.backoffSlotSize = backoffSlotSize;
+        this.random = random;
```

Comment:
s/random/jitter/ - it's fine for this to always return 1 in testing, and be a random implementation in prod?

Reply:
It's important to get jitter in our simulations as well, otherwise the we may cause waves of requests. In tests using a constant is fine, but ideally we don't measure wall clock time in tests and watch the request rate/etc.